imap/message.c

```diff
@@ -3134,9 +3134,9 @@ static int message_read_body(struct protstream *strm, struct body *body, const c
 
         body->type = xstrdup("MULTIPART");
         do {
-            body->subpart =
-                (struct body *)xrealloc((char *)body->subpart,
-                                        (body->numparts+1)*sizeof(struct body));
+            body->subpart = xzrealloc(body->subpart,
+                                      body->numparts * sizeof(struct body),
+                                      (body->numparts + 1) * sizeof(struct body));
             memset(&body->subpart[body->numparts], 0, sizeof(struct body));
```

Comment:
That's not required anymore, right?

Reply:
Right

Reply:
Hah, yeah, forgot to delete it after changing the line above.  The simpler or more repetitive the change, the dumber the mistakes 😅